### PR TITLE
Working Example

### PR DESCRIPTION
**Pull request by owensims1**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,32 +30,28 @@ from ionic_langchain.tool import IonicTool
...
-    memory=memory,
+    llm=ChatOpenAI(openai_api_key="your_key_here", temperature=0.7),
+    agent=AgentType.CHAT_ZERO_SHOT_REACT_DESCRIPTION,
     handle_parsing_errors=True,
     verbose=True,
 )
 
-agent.run(input="Roadhouse VHS")
+
+input = "Where can I get tide pods"
```
Comment (gmkohler) [inline README.md:48]: hiding somewhere in your child's room

File: README.md [hunk 2]
```diff
@@ -30,32 +30,28 @@ from ionic_langchain.tool import IonicTool
...
-    chat_memory=redis_memory,
-    memory_key="chat_history",
-)
 
 agent = initialize_agent(
     tools=tools,
-    llm=ChatOpenAI(openai_api_key=os.environ.get("OPENAI_API_KEY"),temperature=0.5),
-    agent=AgentType.CHAT_CONVERSATIONAL_REACT_DESCRIPTION,
-    memory=memory,
+    llm=ChatOpenAI(openai_api_key="your_key_here", temperature=0.7),
```
Comment (gmkohler) [inline README.md:41]: should we change our impl?

File: README.md [hunk 3]
```diff
@@ -30,32 +30,28 @@ from ionic_langchain.tool import IonicTool
...
 )
 
-agent.run(input="Roadhouse VHS")
+
+input = "Where can I get tide pods"
+
+agent.run(input=input)
 ```
+
+_Please see the [langchain agent docs](https://python.langchain.com/docs/modules/agents/) for more details on how to build and run agents_
```
Comment (gmkohler) [inline README.md:53]: underscore?

**Review by gmkohler — APPROVED** [review]
do we want the underscore in `_Please` on line 53?

File: README.md (hunk 3 above)
Comment (gmkohler) [reply README.md:53]: oh that's italicization, carry on

File: README.md (hunk 1 above)
Comment (gmkohler) [reply README.md:48]: `Tide PODS®`

File: README.md (hunk 2 above)
Comment (owensims1) [reply README.md:41]: in our agent? no that works because it has redis, etc.

**Comment by gmkohler** [comment]
Should we port these changes into Mintlify?

**Pull request merged**